<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note Taker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="NoteTaker.qrc">
    <normaloff>:/images/images/text.png</normaloff>:/images/images/text.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="fSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QWidget" name="fListPane" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="fSearchFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="fSearchLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="NoteTaker.qrc">:/images/images/view.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fSearchEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="fTagSelectorLabel">
           <property name="text">
            <string>&amp;Tags:</string>
           </property>
           <property name="buddy">
            <cstring>fTagsComboBox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fTagsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="fNoteList"/>
       </item>
       <item>
        <widget class="QFrame" name="fSettingsPane">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="fAddButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="NoteTaker.qrc">
              <normaloff>:/images/images/add2.png</normaloff>:/images/images/add2.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fMessageLabel">
            <property name="text">
             <string>Synced</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="fSettingsButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="NoteTaker.qrc">
              <normaloff>:/images/images/gear.png</normaloff>:/images/images/gear.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fEditorPane" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NoteEditor" name="fEditor"/>
       </item>
       <item>
        <widget class="QFrame" name="fTagsPane">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="fTagsLabel">
            <property name="text">
             <string>Tags:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fTagsEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="fFavouriteButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="NoteTaker.qrc">
              <normaloff>:/images/images/star_grey.png</normaloff>:/images/images/star_grey.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="fExportButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="NoteTaker.qrc">
              <normaloff>:/images/images/export1.png</normaloff>:/images/images/export1.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NoteEditor</class>
   <extends>QTextEdit</extends>
   <header>NoteEditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fEditor</tabstop>
  <tabstop>fNoteList</tabstop>
  <tabstop>fSearchEdit</tabstop>
  <tabstop>fAddButton</tabstop>
  <tabstop>fSettingsButton</tabstop>
  <tabstop>fTagsEdit</tabstop>
  <tabstop>fFavouriteButton</tabstop>
  <tabstop>fExportButton</tabstop>
 </tabstops>
 <resources>
  <include location="NoteTaker.qrc"/>
 </resources>
 <connections/>
</ui>
